spring-cloud-stream-samples/pom.xml

```diff
@@ -21,6 +21,7 @@
 		<module>tap</module>
 		<module>double</module>
 		<module>extended</module>
+		<module>rxjava</module>
```

Comment:
I see what you mean. I can move this in a separate project. Infact, I was going down that route initially and decided to use the current approach as we just add rxjava as a `provided` dependency.